edb/lib/ext/auth.edgeql

```diff
@@ -144,6 +144,59 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
                 indicates that the token should never expire.";
             set default := <std::duration>'336 hours';
         };
+
+    };
+
+    create scalar type ext::auth::SMTPSecurity extending enum<PlainText, TLS, STARTTLS, STARTTLSOrPlainText>;
```

Comment:
Should it be `StartTLS`?

Reply:
Yeah I'm seeing both in different places. [RFC 3207](https://www.ietf.org/rfc/rfc3207.txt) uses "STARTTLS".